analyses/cms-open-data-ttbar/ttbar_analysis_pipeline.py

```diff
@@ -192,9 +194,12 @@ def process(self, events):
         else:
             xsec_weight = 1
 
-        # setup triton gRPC client
-        if self.use_inference and self.use_triton:
-            triton_client = utils.clients.get_triton_client(utils.config["ml"]["TRITON_URL"])
+        # setup triton gRPC client or xgboost
+        if self.use_inference:
+            if self.use_triton:
+                triton_client = utils.clients.get_triton_client(utils.config["ml"]["TRITON_URL"])
+            else:
```

Comment:
Is this some leftover from debugging xgboost?

Reply:
? This is mandatory if you use the `FuturesExecutor` : the processes will lock if you attempt to share an xgboost model from the originating processes, and best practice suggests they be loaded inside the daughters anyway. For Dask this should have no meaningful effect since there are no copy-on-write benefits to be had anyway.

Reply:
I see, we probably haven't run that executor in quite a while then. Thanks!